cumulusci/core/sfdx.py

```diff
@@ -15,7 +16,7 @@ def sfdx(
     username=None,
     log_note=None,
     access_token=None,
-    args=None,
+    args: T.Optional[T.List[str]] = None,
```

Comment:
Is there a specific reason you aren't importin `Optional` and `List` directly from `typing`? That's how I've seen it in our codebase for he most part, so just curious here.

Reply:
I am -- I imported typing as T so that each individual name doesn't need to be imported separately. It's an idiom @prescod has started using elsewhere.

Reply:
Yeah it just gets tedious maintaining the import list. "No unions used in this file anymore, need to delete it. Now I need Optional. Need to add it."

Really, this is the kind of thing that import * was invented for but the taboo against it is so strong in the Python world that I don't dare buck the trend! (and linters would probably hate it)